### PR TITLE
Add AlphaWallet connector

### PR DESCRIPTION
**Pull request by JamesSmartCell**
Adding AlphaWallet as an option for onboard interface.

### CODE REVIEW

**Comment by taylorjdawson** [comment]
@JamesSmartCell thanks for this! Taking a look 👀

File: src/modules/select/wallets/alphawallet.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    wallet: async (helpers: Helpers) => {
+      const { getProviderName, getAddress, getNetwork, getBalance } = helpers
+      const alphawalletProvider =
+        (window as any).ethereum ||
+        ((window as any).web3 && (window as any).web3.currentProvider)
+
+      const isAlphaWallet = getProviderName(alphawalletProvider) === 'AlphaWallet'
+      let createProvider
+
+      if isAlphaWallet && rpcUrl) {
```
Comment (taylorjdawson) [inline src/modules/select/wallets/alphawallet.ts:26]: @JamesSmartCell Should there be an opening parenthesis here? 
```suggestion
      if (isAlphaWallet && rpcUrl) {
```

**Review by taylorjdawson — CHANGES_REQUESTED** [review]

File: src/modules/select/wallet-icons/icon-alphawallet.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            </g>
+            <path d="M23.4685844,10.517303 C23.4685844,11.7630036 22.4581892,12.7733509 21.2124886,12.7733509 C19.966788,12.7733509 18.9569677,11.7630036 18.9569677,10.517303 C18.9569677,9.2716503 19.966788,8.26125509 21.2124886,8.26125509 C22.4581892,8.26125509 23.4685844,9.2716503 23.4685844,10.517303" id="Fill-1" fill="#FFD400"></path>
+        </g>
+    </g>
+</svg>
+`
+
+export default alphawalletIcon
```
Comment (taylorjdawson) [inline src/modules/select/wallet-icons/icon-alphawallet.ts:27]: @JamesSmartCell Add new line here

**Review by taylorjdawson — CHANGES_REQUESTED** [review]
Please run the following command to format your code according to the formatting guidelines
```
npx prettier --write src/modules/select/wallets/ownbit.ts
```

**Comment by taylorjdawson** [comment]
@JamesSmartCell Also if you could update this branch with develop, thanks!

**Comment by JamesSmartCell** [comment]
Hi @taylorjdawson thanks for checking our PR :) I merged and applied formatting as directed, can you re-check? Thanks.

**Comment by taylorjdawson** [comment]
Looks good! 👍

**Review by taylorjdawson — APPROVED** [review]

**Pull request merged**